crates/forge/tests/cli/inline_config.rs

```diff
@@ -147,14 +147,14 @@ forgetest!(invalid_value, |prj, cmd| {
 Compiler run successful!
 
 Ran 1 test for test/inline.sol:Inline
-[FAIL: invalid type: found sequence, expected u32 for key "default.runs.fuzz" in inline config] test(bool) ([GAS])
+[FAIL: invalid type: found sequence, expected u32 for key "default.fuzz.runs" in inline config] setUp() ([GAS])
```

Comment:
I guess `extract_inner` flipped the keys? Lol
Anyway it's gone now

Reply:
:) oops, missed in review (it affects err msg only so not a biggie)